scst-scan/ivs-create-your-own.hbs.md

```diff
@@ -64,11 +64,21 @@ To customize an ImageVulnerabilityScan to use your scanner:
 
     Because volumes on a Tekton pipeline are shared amongst steps, files created by one step are consumable by the other steps. The scan controller applies the following security context to `pipelinerun.spec.podTemplate`:
 
-  ```console
-  runAsUser: 65534
-  fsGroup: 65534
-  runAsGroup: 65534
-  ```
+    ```console
+    runAsUser: 65534
+    fsGroup: 65534
+    runAsGroup: 65534
+    ```
+
+    > **Note** If you populate any of the following fields in the `securityContext`, you must
+    populate all the other fields or you may see a runtime error in the `ImageVulnerabilityScan` controller:
+    ```
+    allowPrivilegeEscalation
+    runAsNonRoot
+    seccompProfile
+    capabilities
+    ```
+    Failing to populate
```

Comment:
Is this line supposed to be here.  Or is this maybe not ready to review?

Reply:
That line shouldn't be there, I'll remove it now.